src/utils.rs

```diff
@@ -1,12 +1,46 @@
 extern crate base64;
 
 use self::base64::encode;
+use http::retrieve_asset;
 use regex::Regex;
+use std::collections::HashMap;
 use url::{ParseError, Url};
 
+/// This monster of a regex is used to match any kind of URL found in a CSS
+/// stylesheet.
+///
+/// There's roughly three different categories that a found url could fit into:
```

Comment:
I think it's better to use "There are" here because there're plural categories listed below